src/cider/nrepl/middleware/debug.clj

```diff
@@ -655,6 +665,10 @@ this map (identified by a key), and will `dissoc` it afterwards."}
                 (eval form1)))
           (throw e))))))
 
+(def ^:dynamic *debug-data-readers*
+  "Reader macros like #dbg which cause code to be instrumented when present."
+  '#{dbg exn dbgexn break light})
```

Comment:
Aagh, somehow I didn't pay attention during a merge conflict and this exn->break! renaming got lost in the process...  @bbatsov do you mind fixing it up on your side?

Reply:
I'll fix it.

Reply:
Done.